browser/ui/webui/commands_ui.cc

```diff
@@ -94,6 +120,15 @@ BraveBrowserWindow* CommandsUI::GetWindow() {
   return static_cast<BraveBrowserWindow*>(GetBrowser()->window());
 }
 
+void CommandsUI::NotifyCommandChanged(int command_id) {
```

Comment:
nit:

```suggestion
void CommandsUI::NotifyCommandChanged(uint_32 command_id) {
```

Reply:
Moved to `int` everywhere instead, as that's what command ids are defined as - for some reason I'd thought they were `uint32s`